website/docs/docs/collaborate/dbt-explorer-faqs.md

```diff
@@ -0,0 +1,176 @@
+---
+title: "dbt Explorer FAQs"
+sidebar_label: "dbt Explorer FAQs"
+description: "Learn more with the FAQs about dbt Explorer, how it works, how to interact with it, and more."
+pagination_next: null
+---
+
+[dbt Explorer](/docs/collaborate/explore-projects) is dbt Cloud’s new knowledge base and lineage visualization experience. It offers an interactive and high-level view of your company’s entire data estate, where you can do dive deeps for the context you need to understand and improve lineage so your teams can trust the data they’re using to make decisions.
+
+## Overview
+
+<expandable alt_header="How does dbt Explorer help with data quality?" >
+
+dbt Explorer makes it easy and intuitive to understand your entire lineage &mdash; from data source to the reporting layer &mdash; so you can troubleshoot, improve, and optimize your pipelines. With built-in features like project recommendations and model performance analysis, you can be sure you have appropriate test and documentation coverage across your estate and quickly spot and remediate slow-running models. With column-level lineage, you can quickly identify the potential downstream impacts of table changes or work backwards to quickly understand the root cause of an incident. dbt Explorer gives teams the insights they need to improve data quality proactively, ensuring pipelines stay performant and data trust remains solid.
+
+</expandable>
+
+<expandable alt_header="How is dbt Explorer priced?" >
+
+With the current Preview, dbt Explorer is available to everyone on the dbt Cloud Enterprise and Team plans. Certain features within dbt Explorer, such as multi-project lineage and column-level lineage, are only available on the Enterprise plan. dbt Explorer can be accessed by users with developer and read-only seats during the Preview period.
+
+</expandable>
+
+<expandable alt_header="What happened to dbt Docs?" >
+
+dbt Explorer will become the default documentation experience for dbt Cloud customers. dbt Docs is still available but doesn't offer the same speed, metadata, or visibility as dbt Explorer and will become a legacy feaure.
```

Comment:
When is this document supposed to ship? Should we change the future tense here so that it's a little more bulletproof?
```suggestion
dbt Explorer is the default documentation experience for dbt Cloud customers. dbt Docs is still available but doesn't offer the same speed, metadata, or visibility as dbt Explorer and will become a legacy feature.
```